bin/main.ml

```diff
@@ -45,52 +44,48 @@ let common_commands_synopsis =
     ; "init project NAME [PATH] [--libs=l1,l2 --ppx=p1,p2 --inline-tests]"
     ]
 
-let default =
+let info =
   let doc = "composable build system for OCaml" in
-  let term =
-    Term.ret
-    @@ let+ _ = Common.term in
-       `Help (`Pager, None)
-  in
-  ( term
-  , Term.info "dune" ~doc
-      ~version:
-        (match Build_info.V1.version () with
-        | None -> "n/a"
-        | Some v -> Build_info.V1.Version.to_string v)
-      ~man:
-        [ `Blocks common_commands_synopsis
-        ; `S "DESCRIPTION"
-        ; `P
-            {|Dune is a build system designed for OCaml projects only. It
+  Cmd.info "dune" ~doc
+    ~version:
+      (match Build_info.V1.version () with
+      | None -> "n/a"
+      | Some v -> Build_info.V1.Version.to_string v)
+    ~man:
+      [ `Blocks common_commands_synopsis
+      ; `S "DESCRIPTION"
+      ; `P
+          {|Dune is a build system designed for OCaml projects only. It
               focuses on providing the user with a consistent experience and takes
               care of most of the low-level details of OCaml compilation. All you
               have to do is provide a description of your project and Dune will
               do the rest.
             |}
-        ; `P
-            {|The scheme it implements is inspired from the one used inside Jane
+      ; `P
+          {|The scheme it implements is inspired from the one used inside Jane
               Street and adapted to the open source world. It has matured over a
               long time and is used daily by hundreds of developers, which means
               that it is highly tested and productive.
             |}
-        ; `Blocks Common.help_secs
-        ; Common.examples
-            [ ("Initialise a new project named `foo'", "dune init project foo")
-            ; ("Build all targets in the current source tree", "dune build")
-            ; ("Run the executable named `bar'", "dune exec bar")
-            ; ("Run all tests in the current source tree", "dune runtest")
-            ; ("Install all components defined in the project", "dune install")
-            ; ("Remove all build artefacts", "dune clean")
-            ]
-        ] )
+      ; `Blocks Common.help_secs
+      ; Common.examples
+          [ ("Initialise a new project named `foo'", "dune init project foo")
+          ; ("Build all targets in the current source tree", "dune build")
+          ; ("Run the executable named `bar'", "dune exec bar")
+          ; ("Run all tests in the current source tree", "dune runtest")
+          ; ("Install all components defined in the project", "dune install")
+          ; ("Remove all build artefacts", "dune clean")
+          ]
+      ]
+
+let cmd = Cmd.group info all
 
 let () =
   Colors.setup_err_formatter_colors ();
   try
-    match Term.Group.eval default all ~catch:false with
```

Comment:
I tested all return values of `eval_value` and it looks equivalent but I'm not completely confident about this